### PR TITLE
Add int and float management for cleanQuantity script and add tests

### PR DESCRIPTION
**Pull request by nriss**
## Description

If a value is saved in int or float, it will not be modified by cleanQuantity and won't return an error.
Addition of cleanQuantity tests

### CODE REVIEW

File: scripts/custom/clean_quantity.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 def clean_quantity(raw_input):
     """Removes input not conform to FHIR quantity type
     """
+    if isinstance(raw_input, float) or isinstance(raw_input, int):
```
Comment (LaRiffle) [inline scripts/custom/clean_quantity.py:9]:
```suggestion
    if isinstance(raw_input, (float, int)):
```

**Review by LaRiffle — APPROVED** [review]

**Review by elsiehoffet-94 — CHANGES_REQUESTED** [review]
Don't forget to change the version in setup.py

**Review by Jasopaum — COMMENTED** [review]
I'm not sure this script is still needed as it basically casts inputs to float and this work is already done by river when needed.
I'm not saying we should totally remove it because some mappings rely on though...

**Review by simonvadee — COMMENTED** [review]
as @elsiehoffet-94 said, fon't forget the version.
I agree with @Jasopaum, if the sql column is already of type int/float, the script clean_quantity should not be used since river does not cast the value as a string.

**Comment by nriss** [comment]
> river does not cast the value as a string.

I suggest to change the description of the cleaning script
`Removes input not conform to FHIR quantity type`
To
`[deprecated: river parses types automatically] Removes input not conform to FHIR quantity type`

**Review by simonvadee — APPROVED** [review]

**Review by elsiehoffet-94 — APPROVED** [review]

**Pull request merged**